network-api/networkapi/wagtailpages/pagemodels/buyersguide/article_page.py

```diff
@@ -90,6 +92,8 @@ class BuyersGuideArticlePage(
 
     def get_context(self, request: http.HttpRequest, *args, **kwargs) -> dict:
         context = super().get_context(request, *args, **kwargs)
+        language_code = get_language_from_request(request)
+        context['categories'] = get_categories_for_locale(language_code)
```

Comment:
I believe we have this on every class for the buyer's guide now, right? What do you think about moving this into a method on a mixin and have each buyer's guide page type inherit from that mixin. Then we could use `page.get_categories` in the template.

Reply:
https://github.com/mozilla/foundation.mozilla.org/issues/9622 Ticket created for this task since this is taking more time then originally anticipated